developers/contributing/README.md

```diff
@@ -49,3 +50,20 @@ If a pull request is a work in progress, please start the title with `[WIP]`. It
 Changes to BonnyCI should follow a design document. To propose a design document, open a pull requests to the [lore repository](https://github.com/BonnyCI/lore). BonnyCI contributors will review and debate the proposed document via pull request comments until the document is accepted. When the pull request is merged, development can begin.
 
 Design documents exist in the [design documentation](designs) directory.
+
+## Adding Projects
```

Comment:
Can we rename this to something like "Adding BonnyCI Projects" to distinguish it from "Adding Projects to BonnyCI"? I initially thought this was about enabling BonnyCI testing on arbitrary projects, but I suppose that would belong in `end_users/` rather than `contributors/`.